src/data/int/gcd.lean

```diff
@@ -118,4 +118,36 @@ dvd.elim H (λl H1, by rw mul_assoc at H1; exact ⟨_, eq_of_mul_eq_mul_left k_n
 theorem dvd_of_mul_dvd_mul_right {i j k : ℤ} (k_non_zero : k ≠ 0) (H : i * k ∣ j * k) : i ∣ j :=
 by rw [mul_comm i k, mul_comm j k] at H; exact dvd_of_mul_dvd_mul_left k_non_zero H
 
+/- A few useful lemmas, mostly generalizations of the same for nat -/
+lemma gcd_comm (i j : ℤ): gcd i j = gcd j i := by { unfold gcd, rw nat.gcd_comm }
+
+lemma gcd_dvd_left (i j : ℤ) : (gcd i j : ℤ) ∣ i :=
+begin
+  apply nat_abs_dvd_abs_iff.mp,
+  change gcd i j ∣ nat_abs i,
```

Comment:
You should be able to remove the `change` line here

Reply:
turns out these were already in gcd_doamin...